Dockerfile

```diff
@@ -1,15 +1,17 @@
 FROM golang:1.21-alpine AS build
 
-LABEL MAINTAINER = 'Friends of Go (it@friendsofgo.tech)'
+LABEL MAINTAINER='Friends of Go (it@friendsofgo.tech)'
 
 ARG TARGETOS=linux
 ARG TARGETARCH=amd64
+ARG TAG=""
 
 RUN apk add --update git
 RUN apk add ca-certificates
 WORKDIR /go/src/github.com/friendsofgo/killgrave
 COPY . .
-RUN go mod tidy && TAG=$(git describe --tags --abbrev=0) \
+RUN go mod tidy \
```

Comment:
I'd prefer if we can leave those changes that aren't strictly needed as part of this pull request, apart.